larpandoracontent/LArControlFlow/NeutrinoIdTool.cc

```diff
@@ -255,13 +256,33 @@ void NeutrinoIdTool<T>::SelectPfosByProbability(const pandora::Algorithm *const
         {
             object_creation::ParticleFlowObject::Metadata metadata;
             metadata.m_propertiesToAdd["NuScore"] = nuProbability;
+	    
+            if (m_persistFeatures)
```

Comment:
This will need a run through clang format.

Reply:
Will do!